mcs/class/System.Net.Http/corefx/NetEventSource.Http.cs

```diff
@@ -0,0 +1,57 @@
+using System.Diagnostics.Tracing;
+using System.Net.Http;
+
+namespace System.Net
+{
+	internal sealed partial class NetEventSource : EventSource
+	{
+		public static readonly NetEventSource Log = new NetEventSource ();
+
+		public static new bool IsEnabled => false;
```

Comment:
This should not be needed and could break linker

Reply:
The property is required - but we might be able to change it's definition into

```
public static NetEventSource Log => throw new PlatformNotSupportedException ();
```

Would that help?

Reply:
Looks like it's passing the tests, so I'm going to change that.